java/src/main/java/com/logicalclocks/hsfs/engine/FeatureViewEngine.java

```diff
@@ -58,8 +58,7 @@ private List<TrainingDatasetFeature> makeLabelFeatures(List<String> labels) {
 
   public FeatureView update(FeatureView featureView) throws FeatureStoreException,
       IOException {
-    FeatureView featureViewUpdated = featureViewApi.update(featureView);
-    featureView.setDescription(featureViewUpdated.getDescription());
```

Comment:
why remove this?

Reply:
Because `featureView` should have the updated description and the backend gets updated. So it is redundant.